### DISCUSSION TITLE
Domain suspended by Google

### DISCUSSION THREAD

**Discussion by Dubz** [dani-garcia/vaultwarden#3046] [Q&A] [answered]
### Subject of the issue

So Chrome recently started giving malware warnings on my self hosted instance of Vaultwarden. I submitted a report stating this was false, and the page was restored roughly the same day.
A week goes by, and now I'm receiving an alert from Google Domains registrar stating I'm hosting phishing content and that it needs to be removed. This is the only thing running (on a subdomain, mind you), so it's hard to see this not being the issue.

### Deployment environment

* vaultwarden version: Latest (Unknown)

* Install method: Google Cloud/Docker instance

* Clients used: ~10

* Reverse proxy and version: Cloudflare

* MySQL/MariaDB or PostgreSQL version: Latest

* Other relevant details: Domain has been turned off and the site is no longer accessible so I am unable to obtain the necessary information.

My only question/concern is does Vaultwarden's login page need to be adjusted further to signify a bigger difference from Bitwarden, or is this how Bitwarden's self hosted instance normally looks? If it's how it normally looks, then this is quite irritating that anything even happened. Not even sure why it came up to begin with.

**Comment by BlackDex** [comment]
Not sure which version you are running, but the current latest has Vaultwarden logos and title.

What normally is the issue here, is that you have your subdomain named `bitwarden`. At least, that is what all the other reports had in common, or some derivative from it.

There isn't anything we can do regarding this. We don't know the algorithms they use to detect this and thus can't adjust anything.

Also, official self hosted Bitwarden instances would have the same issue, since they use the exact same web-vault as the cloud solution.

**Reply by Dubz** [reply]
> Not sure which version you are running, but the current latest has Vaultwarden logos and title.
> 
> What normally is the issue here, is that you have your subdomain named `bitwarden`. At least, that is what all the other reports had in common, or some derivative from it.
> 
> There isn't anything we can do regarding this. We don't know the algorithms they use to detect this and thus can't adjust anything.
> 
> Also, official self hosted Bitwarden instances would have the same issue, since they use the exact same web-vault as the cloud solution.

Thanks for the information.

You are correct that I used bitwarden in the subdomain, but I did also forget about the logo being different now let alone the footer text added.
I guess now all I can do is wait and plan to move hosting off of Google entirely if they can't assist.

Thanks!

EDIT:
I'll be sure to give a progress update if/when I hear back (hopefully soon...).

**Comment by Dubz** [comment, marked as answer]
So I reached out to Google Domains support, again, and finally got a response from them. Their response, and I quote:

> We have reviewed the domain and based on further investigation the domain is no longer resolving to unsafe links, therefore the domain suspension has been lifted. To avoid this kind of issue, please review and make sure to comply with our [Terms of Service](https://domains.google.com/tos).

When asking for further information (as in, what the link is they are referring to), they stated the following:

> There was no specific URL reported but what was reported in general was your domain **REDACTED**. After the report we did a safebrowing check and confirmed it results in unsafe content at that time which triggered the suspension.
But we have unsuspended now since we can confirm there's no longer unsafe content related to the domain.

1. I'm not sure how they possibly ran a safebrowsing check on the domain, given that it was not even resolving records being in its suspended state.
2. I didn't make any changes from the time it was shut down to the time it was restored. Unless they assumed nothing changed from the first detection and never revalidated before suspending, then they're full of crap. I did update Vaultwarden instance within the last few days, so that could possibly have been related (although unlikely).
3. Don't use Google Domains for your registrar apparently...

In sight of all this, I've opted to transfer all of my domains off of Google Domains and over to Porkbun.com after finding multiple references to them online as being great. My main registrar is Cloudflare, however some domains are not currently supported and require an alternative registrar (now Porkbun).

Now not only do I no longer have to worry about these shenanigans pulled by Google, but I'm saving money with cheaper registration fees.

**Reply by esackbauer** [reply]
Porkbun is great, have many domains there since years without problem.
It is also supported by [ddns-updater](https://github.com/qdm12/ddns-updater)
